packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -414,35 +417,25 @@ impl DatabaseApi<'_> {
     ) -> Result<PersistedLink, QueryError> {
         Ok(self
             .store
-            .get_links(&StructuralQuery {
-                expression: Expression::All(vec![
-                    Expression::for_link_by_source_entity_id(source_entity_id),
-                    Expression::Eq(vec![
-                        Expression::Path(Path {
-                            segments: vec![
-                                PathSegment {
-                                    identifier: "type".to_owned(),
-                                },
-                                PathSegment {
-                                    identifier: "baseUri".to_owned(),
-                                },
-                            ],
-                        }),
-                        Expression::Literal(Literal::String(link_type_id.base_uri().to_string())),
-                    ]),
-                    Expression::Eq(vec![
-                        Expression::Path(Path {
-                            segments: vec![
-                                PathSegment {
-                                    identifier: "type".to_owned(),
-                                },
-                                PathSegment {
-                                    identifier: "version".to_owned(),
-                                },
-                            ],
-                        }),
-                        Expression::Literal(Literal::Float(link_type_id.version() as f64)),
-                    ]),
+            .get_links(&NewStructuralQuery {
+                filter: Filter::Any(vec![
```

Comment:
Should this be `Filter::All`? Otherwise we'd match on just the version of the type ID, right?

Reply:
Yes, thanks for spotting 
```suggestion
                filter: Filter::All(vec![
```